vhdbuilder/packer/write-release-notes-windows.ps1

```diff
@@ -97,10 +97,14 @@ if ($env:containerRuntime -eq 'docker') {
     LOG (docker images --format='{{json .}}' | ConvertFrom-Json | Format-Table Repository, Tag, ID)
 } else {
     Log "ContainerD Info"
+    # starting containerd for printing containerD info, the same way as we pre-pull containerD images in configure-windows-vhd.ps1
+    Start-Job -Name containerd -ScriptBlock { containerd.exe }
     $containerDVersion = (ctr.exe --version) | Out-String
     Log ("Version: {0}" -f $containerDVersion)
     Log "Images:"
     LOG (ctr.exe -n k8s.io image ls)
```

Comment:
What is the difference for Log and LOG?

Reply:
Powershell is not case sensitive for the function name. The script is mixing `Log` and `LOG`, and I did not realize this in this change but planned to use one pattern months ago.